### ISSUE TITLE
Spaces are there but take up no space in comments.

### ISSUE THREAD

**Issue by jcklpe** [Microsoft/vscode#59587]
- VSCode Version: 1.27.2
- OS Version: Windows 10 1803

Steps to Reproduce:

1. type out a comment in any language

Does this issue occur when all extensions are disabled?: No, but I believe the responsible extension will not fix this issue by being manually disabled. 

Okay so here's the deal. I know that my particular set up is very odd. It's a highly customized theme, using a bunch of different extensions, and custom js/css injected into the application through an extension. 

BUT I think it's worth reporting because it may help the vscode devs or extension devs think of better more anti-fragile ways to approach development. 

Here's the main problem. 

![image](https://user-images.githubusercontent.com/9451726/46182999-704f6580-c294-11e8-926d-e1c93898a8dc.png)
All my comments in all languages are not displaying the spaces. The spaces ARE THERE. I know this because if I write "Project     X" then after backspacing X it will take 4 more backspaces before it backspaces the t in "Project". 

I think part of the problem is that I have used the JS/CSS injection extension (found [[here]](https://marketplace.visualstudio.com/items?itemName=be5invis.vscode-custom-css) )to change the font of comments. Basically the way I do this is by targeting parts of the text editor text by their CSS class names instead of their tokentheme names. It's a hacky solution but I think It's useful and an official solution should be supported in vscode. ( https://github.com/Microsoft/vscode/issues/23707 )

![image](https://user-images.githubusercontent.com/9451726/46183302-fc15c180-c295-11e8-96bc-3c5960f3ee66.png)

As an official solution does not seem to be on the horizon I'm trying to figure out what is going on here. It was working just fine just a few weeks ago. You can see my comments in that above issue showing it working just fine. So what happened? 

Additionally when I disable the load CSS/JS extension, it still applies the injected styles to my comments. So therefore I need to open up the styles, while the extension is turned on and erase then, then save, right? 

Well for those changes to actually take place I need to reload the JS and CSS. Unfortunately.... it seems I can't do that. If I try, it tells me I need to open the program as an admin. I right click and say "open as admin" and.... it opens no problem but I still get the same "you need to be admin to reload these changes". I tried forcing it to always open the program as an admin, and.... when I do I get this:

![image](https://user-images.githubusercontent.com/9451726/46182987-62014980-c294-11e8-8e6d-f3977f4f7302.png)

So what's there left for me to do here? 

I have no clue why it's having trouble with admin stuff. I thought maybe it was some weird permissions issue but I gave all users full permissions to the exe using Window's security settings etc.

**Comment by rebornix** [comment]
This issue was initially triggered by vscode-custom-css extension which will inject styles into our code base. But when users disabled this extension and tried to revert back the change this extension made, the window crashes. I'd like to have @joaomoreno 's expertise on this.

**Comment by jcklpe** [comment]
@rebornix 

I have an update. I found that the reason it was was crashing is because there were other instances of vscode open in the background on a different screen. So it appears that vscode can't open in admin mode if there is another window of it open elsewhere. 

Once I figured this out, I was able to close all other windows and edit my custom css injection like normal. The initial problem with my comments having invisible spaces was fixed by me switching to a different font. I don't know why the original font (called Aller Sans) wasn't working but switching to any other installed font fixed the issue. switching back to Aller Sans made the issue reappear. I ended up just switching to using a font called Sanchez Slab and I've been happy with the results since. 

I don't consider this to be an issue for me anymore but I can leave this issue open if you guys think it's important or helps your improving the robustness of vscode etc. 

Thank you for responding though!
